### PR TITLE
Restoring Sapienza faculty apparently not included in latest merge

### PR DESCRIPTION
**Pull request by cdc08x**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [X] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -15923,8 +15956,11 @@ Robert van Engelen,Florida State University,https://www.asee.org/public/conferen
 Robert van Liere,CWI,http://homepages.cwi.nl/~robertl,NOSCHOLARPAGE
 Roberta Gori,University of Pisa,http://www.di.unipi.it/~gori,NOSCHOLARPAGE
 Roberta Sinatra,IT University of Copenhagen,https://www.robertasinatra.com,PL8nGh4AAAAJ
+Roberto Baldoni,Sapienza University of Rome,http://www.diag.uniroma1.it/users/roberto%20baldoni,82tR6VoAAAAJ
+Roberto Beraldi,Sapienza University of Rome,http://www.diag.uniroma1.it/users/roberto%20beraldi,WaJQ4-wAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:15960]: Please remove the incorrect entry (not sure which).

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please remove spurious duplicate.

**Comment by cdc08x** [comment]
Hi!

I am not sure if that is the issue but if this is the one:
`Roberto Baldoni,Sapienza University of Rome,http://www.diag.uniroma1.it/users/roberto%20baldoni,82tR6VoAAAAJ
Roberto Beraldi,Sapienza University of Rome,http://www.diag.uniroma1.it/users/roberto%20beraldi,WaJQ4-wAAAAJ`
actually,
Roberto Ba_ldoni_
and
Roberto Be_raldi_
are two different people. Is it OK as a clarification?

Thanks a lot for your great, admirable work!

Cheers

**Comment by emeryberger** [comment]
Sigh, somehow I saw those as the same - sorry! Thanks!

**Review by emeryberger — APPROVED** [review]

**Comment by cdc08x** [comment]
Never mind! No problem at all. Thanks a lot for your help!

**Pull request merged**